mod/carrierwave/lib/carrier_wave/card_mount.rb

```diff
@@ -29,13 +29,13 @@ def mount_uploader column, uploader=nil, options={}, &block
           remove_#{column}!
         end
 
-        event :mark_remove_#{column}_false_event, :finalize, on: :update do
-          mark_remove_#{column}_false
-        end
+        # event :mark_remove_#{column}_false_event, :finalize, on: :update do
+        #   mark_remove_#{column}_false
+        # end
 
-        event :store_previous_changes_for_#{column}_event, :store,
+        event :reset_previous_changes_for_#{column}_event, :store,
               on: :update, when: proc { |c| !c.history? } do
-          store_previous_changes_for_#{column}
+          reset_previous_changes_for_#{column}
```

Comment:
There is not longer a "store_previous_changes" method in mount.rb. Maybe they renamed it to this because that's more accurate? Or maybe it's something different?

Reply:
It's something different: https://github.com/carrierwaveuploader/carrierwave/commit/aac25c10af4218d6e1e70f90154b847b54ce0334